docs/conf/portland/2020/news/schedule.rst

```diff
@@ -0,0 +1,59 @@
+:template: {{year}}/generic.html
+
+.. post:: July 17, 2020
+   :tags: {{year}}, portland-{{year}}, schedule
+
+Announcing Full Schedule
+========================
+
+Hey everyone!
+
+We're less than a month away from the first ever online Portland conference and we're working hard to make sure it'll be as welcoming and as fun as the in-person event you're probably familar with.
+
+Today we're announcing the full presentation schedule and the onnline platform we'll be hosting the conference on.
+
+Conference platform
+--------------------
+
+The conference will be held on https://hopin.to/, which not only provides a stage for our amazing speakers, but also has integrated chat, unconference rooms with breakout calls, and impromptu 1-on-1 meetings. We're updating our welcome guide so it'll all feel familar before you've even arrived.
+
+Full presentation schedule
+--------------------------
+
+The conference website is now updated with the `talk schedule <https://www.writethedocs.org/conf/portland/{{year}}/schedule/>`_ for the main stage presentations, so you can start planning your time (and getting excited!) for the conference. We're excited to have Q&A after most talks
+
+As always, we'll be running the `unconference <https://www.writethedocs.org/conf/portland/{{year}}/unconference/>`_, and `lightning talks <https://www.writethedocs.org/conf/portland/2019/lightning-talks/>`_ each day after lunch.
+
+You'll be able to sign up for unconference sessions and lightning talks during the conference.
+
+
+Start thinking about Writing Day
+--------------------------------
+
+Our pre-conference `Writing Day <https://www.writethedocs.org/conf/portland/2019/writing-day/>`_ will happen on Sunday, and it is much more fun when folks come with their own projects in mind!
+
+If you have a project or are part of a community that you think would benefit from a writing sprint, start brainstorming now about what you might want to focus on.
+If you'd like us to include your project on the Writing Day page, drop us an email at `portland@writethedocs.org <mailto:portland@writethedocs.org>`_.
+Or just show up on Sunday ready to introduce your project!
+
+Shirts on sale
+--------------
+
+The Write the Docs Portland 2019 Pop-Up Shop is now open! We have loose and fitted options from different fabrics, and you can choose exactly the size and cut you want!
+
+The campaign will run in 3-day batches until TODO.
```

Comment:
```suggestion
The campaign will run in 3-day batches until August 30th. Shipping is available from the USA or Europe. However, due to the global shipping situation, we recommend you order your shirt at least 2-3 weeks in advance if you want to sport your new threads during the conference.
```

Reply:
Got a link?

Reply:
Damn, I thought I added it to the suggestion. Here it is again: https://teespring.com/wtd-portland-2020